src/systemctl/systemctl.c

```diff
@@ -6317,20 +6329,29 @@ static int log_setting_internal(sd_bus *bus, const BusLocator* bloc, const char
                 r = bus_set_property(bus, bloc,
                                      level ? "LogLevel" : "LogTarget",
                                      &error, "s", value);
-                if (r < 0)
-                        return log_error_errno(r, "Failed to set log %s of %s to %s: %s",
-                                               level ? "level" : "target",
-                                               bloc->destination, value, bus_error_message(&error, r));
+                if (r < 0) {
+                        log_error_errno(r, "Failed to set log %s of %s to %s: %s",
+                                        level ? "level" : "target",
+                                        bloc->destination, value, bus_error_message(&error, r));
+                        if (sd_bus_error_has_name(&error, SD_BUS_ERROR_UNKNOWN_METHOD))
```

Comment:
hmm, the error returned depends on the bus implementation and app.

Might also be one of SD_BUS_ERROR_UNKNOWN_OBJECT, SD_BUS_ERROR_UNKNOWN_INTERFACE, SD_BUS_ERROR_UNKNOWN_PROPERTY

Reply:
maybe a common bus_error_is_unsupported_api() helper might make sense, or so